src/triedb.rs

```diff
@@ -147,6 +149,52 @@ impl<'db> Trie for TrieDB<'db> {
     }
 }
 
+impl<'db> CryptoStructure for TrieDB<'db> {
```

Comment:
IMO, we need a detailed description of our Proof format.
It may be a comment or a spec document.
@sgkim126 @junha1 What do you think about adding the description of detailed proof format in the comment of the `make_proof` function.

Reply:
In code?

Reply:
Yes, in code. Managing comment is easier than a document.

Reply:
Is it for the light client implementer?

Reply:
I think it's ok to leave it as a new issue

Reply:
I agree that we need to add comments. At least, we should describe how `CryptoProof` is composed.
@junha1 Would you do it in this commit? I think you will do this issue when we create a new issue for it since you implemented it. If then, I think it's better to do it in this commit.

Reply:
Ok I agree. Note that this repo might be given as a reference implementation of our cryptographic proof, for those ICS implementer. (who wants to make a light client of Foundry). In that case we should provide very detailed documentation that makes them be able to develop this in other languages from scratch, starting from what RLP is and so on.
I will put some description in code, but we should consider super-detailed formal specification for this later. (That's why I thought it deserves a new issue)

Reply:
@majecty @sgkim126 Please check! I added comments both for the verification and creation.

Reply:
Could you draw a simple ASCII-art for it? If you add an example of tree, it will be helpful to understand it.